src/parameterizations/vertical/MOM_bulk_mixed_layer.F90

```diff
@@ -3493,12 +3493,12 @@ subroutine bulkmixedlayer_init(Time, G, GV, US, param_file, diag, CS)
                  "layers before sorting when ML_RESORT is true.", &
                  units="nondim", default=0, fail_if_missing=.true.) ! Fail added by AJA.
   ! This gives a minimum decay scale that is typically much less than Angstrom.
-  ustar_min_dflt = 2e-4*US%s_to_T*CS%omega*(GV%Angstrom_m + GV%H_to_m*GV%H_subroundoff)
```

Comment:
is US%s_to_T removed in the new version because the unit conversion is done within the get param call below?

Reply:
Yes, exactly so.